sktime/transformations/series/detrend/tests/test_mstl.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import pandas as pd
+from sktime.datasets import load_airline
+from sktime.transformations.series.detrend import MSTL
```

Comment:
the import goes like this
```suggestion
from sktime.transformations.series.detrend.mstl import MSTL
```